### DISCUSSION TITLE
Reuse of T&Cs and policy text

### DISCUSSION THREAD

**Discussion by mackiexx** [element-hq/element-meta#818] [Q&A] [answered]
### Steps to reproduce

I am asking this in good faith, i am trying to setup my own chat website with element, can i copy and use their privacy policy/terms of use on my website without any issue?

I had no other place to ask.

#### What did you expect?
I am asking to be sure i do not breach the rights given
### Will you send logs?

No

**Comment by daniellekirkwood** [comment]
Hey, thanks for asking. I've requested that our internal team take a look at this and get back to you. We'll be in touch.

**Comment by kittykat** [comment, marked as answer]
@mackiexx the Element docs, especially legal docs, have a lot of things to cover a mix of proprietary and F/LOSS, as well as multiple jurisdictions. We would like to point you at https://matrix.org/legal/privacy-notice/ and https://matrix.org/legal/terms-and-conditions/ , both of which have a note at the end saying that they may be reused.
